zuul.d/ansible-network-jobs.yaml

```diff
@@ -0,0 +1,23 @@
+---
+- job:
+    name: network-ee-integration-tests-ansible-network
+    parent: ansible-network-vyos-appliance
+
+- job:
+    name: network-ee-integration-tests-ansible-network
+    parent: network-ee-integration-tests
+    vars:
+      test_ansible_skip_tags: local
+      collection_namespace: ansible
+      collection_name: network
+      # NOTE(pabelanger): We can only use RSA ssh keys for vyos.
+      test_fips_mode: false
+    nodeset: vyos-1.1.8-python38
+
+- job:
+    name: network-ee-integration-tests-ansible-network-libssh
+    parent: network-ee-integration-tests-ansible-network
+    vars:
+      test_ansible_network_cli_ssh_type: libssh
+      # NOTE(pabelanger): We can only use RSA ssh keys for vyos.
+      test_fips_mode: false
```

Comment:
this can be dropped, we pull it in from network-ee-integration-tests-ansible-network

Reply:
Updated. Thanks